e2e/cypress/integration/dashboard-compressor.spec.ts

```diff
@@ -1,7 +1,7 @@
-const KB = 2 ** 10
-const MB = KB * KB
-
 function humanSizeToBytes (text) {
+  const KB = 2 ** 10
```

Comment:
Should we use @transloadit/prettier-bytes for consistency?

Reply:
But we need the reverse here, from pretty to bytes?

Reply:
We should rename the function to `uglierBytes`.

Reply:
done